terraform/dev/db.tf

```diff
@@ -0,0 +1,45 @@
+resource "aws_docdb_cluster" "db_cluster" {
+  cluster_identifier              = local.namespace
+  engine                          = "docdb"
+  master_username                 = "root"
+  master_password                 = "etsoperations123" # TODO use ssm parameter store
```

Comment:
SSM is easier and one time, easier to maintain than github actions. I prefer SSM since it goes too well with terraform and also local workflow is seamless